### PR TITLE
Add integration specs for components

### PR DESCRIPTION
**Pull request by DawidHaratyk**
- OnboardingProfileModal
- LoadingData
- TokenSetup
- LoadingSkeleton
- SurveyFinishedWrapper

### CODE REVIEW

File: team-fingerprints-web/src/components/LoadingSkeleton/__tests__/LoadingSkeleton.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    render(<LoadingSkeleton />);
+
+    expect(screen.getAllByTestId("skeleton")).toHaveLength(3);
+  });
+
+  it("renders provided number of Skeleton components", () => {
+    render(<LoadingSkeleton count={5} />);
+
+    expect(screen.getAllByTestId("skeleton")).toHaveLength(5);
+  });
```
Comment (tobob) [inline team-fingerprints-web/src/components/LoadingSkeleton/__tests__/LoadingSkeleton.spec.tsx:16]: you can check if proper classnames were applied when `fullWidth` prop is provided

File: team-fingerprints-web/src/components/TokenSetup/__tests__/TokenSetup.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  render(
+    <TokenSetup>
+      <div>content</div>
+    </TokenSetup>
+  );
+};
+
+describe("TokenSetup", () => {
+  it("renders children if tokenPresent is true and saves token in local storage", async () => {
+    const regExp = new RegExp(
```
Comment (tobob) [inline team-fingerprints-web/src/components/TokenSetup/__tests__/TokenSetup.spec.tsx:16]: better name will be `uuid4RegExp`

**Review by tobob — APPROVED** [review]
2 small changes good to have ;) but overall looks good 💪🏼

File: team-fingerprints-web/src/components/LoadingSkeleton/__tests__/LoadingSkeleton.spec.tsx (hunk 1 above)
Comment (DawidHaratyk) [reply team-fingerprints-web/src/components/LoadingSkeleton/__tests__/LoadingSkeleton.spec.tsx:16]: Right, done

File: team-fingerprints-web/src/components/TokenSetup/__tests__/TokenSetup.spec.tsx (hunk 2 above)
Comment (DawidHaratyk) [reply team-fingerprints-web/src/components/TokenSetup/__tests__/TokenSetup.spec.tsx:16]: Right, done

File: team-fingerprints-web/src/components/OnboardingProfileModal/__tests__/OnboardingProfileModal.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+interface RenderComponentProps {
+  history?: MemoryHistory;
+  visibleProfileModal?: boolean;
+}
+
+const renderComponent = ({
+  history,
+  visibleProfileModal = true,
+}: RenderComponentProps) => {
```
Comment (shirve) [inline team-fingerprints-web/src/components/OnboardingProfileModal/__tests__/OnboardingProfileModal.spec.tsx:17]: You can assign empty object here so you don't have to pass it every time you are rendering without any parameters. Makes it look cleaner 😄 
```suggestion
}: RenderComponentProps = {}) => {
```

File: team-fingerprints-web/src/components/OnboardingProfileModal/__tests__/OnboardingProfileModal.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    expect(history.location.state).toEqual({ surveyId: "100" });
+  });
+
+  it("does not close modal if click outside or press escape", async () => {
+    renderComponent({});
+
+    await userEvent.click(document.body);
+    fireEvent.keyPress(global.document, {
+      key: "Escape",
+    });
```
Comment (shirve) [inline team-fingerprints-web/src/components/OnboardingProfileModal/__tests__/OnboardingProfileModal.spec.tsx:62]: You can use `userEvent` for that too
```suggestion
    await userEvent.keyboard("{esc}");
```

**Review by shirve — COMMENTED** [review]
Looks good, just a few small suggestions from me

For the future I would suggest using `faker` for props instead of hardcoding values like `ids`, `titles`, `descriptions` etc.

Also if all your properties are optional and you assign empty object after props...
```
const renderComponent = ({ property }: Props = {}) => ...
```
...you won't have to pass empty parenthesis every time you render without passing any props. Makes it look cleaner 😉

File: team-fingerprints-web/src/components/OnboardingProfileModal/__tests__/OnboardingProfileModal.spec.tsx (hunk 4 above)
Comment (DawidHaratyk) [reply team-fingerprints-web/src/components/OnboardingProfileModal/__tests__/OnboardingProfileModal.spec.tsx:62]: Right, done

File: team-fingerprints-web/src/components/SurveyFinishedWrapper/__tests__/SurveyFinishedWrapper.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+};
+
+const renderComponent = ({
+  children,
+  surveyTitle = "",
+  description = "our description",
+}: RenderComponentProps) => {
+  render(
+    <SurveyFinishedWrapper surveyTitle={surveyTitle} description={description}>
+      {children || <div></div>}
```
[the remark this answers is not in the document]
Comment (DawidHaratyk) [reply team-fingerprints-web/src/components/SurveyFinishedWrapper/__tests__/SurveyFinishedWrapper.spec.tsx:19]: Right, good to know

File: team-fingerprints-web/src/setupTests.ts [hunk 6]
```diff
@@ -11,6 +11,16 @@ jest.mock("@auth0/auth0-react", () => {
   return mockAuth0;
 });
 
+// access to localStorage
+Object.defineProperty(window, "localStorage", {
+  value: {
+    setItem: jest.fn(() => null),
+    getItem: jest.fn(() => null),
+    removeItem: jest.fn(() => null),
```
[the remark this answers is not in the document]
Comment (DawidHaratyk) [reply team-fingerprints-web/src/setupTests.ts:19]: Yes, you're right

**Pull request merged**